jkube-kit/build/service/docker/src/main/java/org/eclipse/jkube/kit/build/service/docker/WatchService.java

```diff
@@ -417,6 +429,7 @@ private WatchMode getWatchMode(ImageConfiguration imageConfig) {
         private String showLogs;
         private Date buildTimestamp;
         private String containerNamePattern;
+        private Runnable postGoalTask;
```

Comment:
You are now using a `Runnable` instead of a `Supplier<Boolean>`.

But the `MavenUtil#callMavenPluginWithGoal` method created specifically for this tasks is still returning a `boolean`.
I'm not sure how important this return value is (why is it ignored in the former vs. why is it returned in the latter).